applicationconfigurations/workflow.md

```diff
@@ -0,0 +1,219 @@
+        - name: message
+          value: "Well hello there"
+
+    - componentName: backend
+```
+
+In this example, the ops config allows a user to override only one of the parameters in the `frontend` component, and it does not allow the user to change any of the `backend` parameters. This allows the author of this ops config to control configuration options for the components therein.
+
+The operator can now deploy the components together with this configuration to create running instances of the components:
+
+```
+$ oamctl install -c ./config.yaml ./frontend.yaml ./backend.yaml -p "message=overridden message!"
+```
+
+### Adding traits to the components
+Traits are attached by application operators. In the next part of this example, traits are added to the two components defined previously.
+
+Our system supports a few different traits:
+
+```console
+$ oamctl trait-list
+╔════════════╤═════════╤═════════════════════════════════════════════╗
+║ NAME       │ VERSION │ PRIMITIVES                                  ║
+╟────────────┼─────────┼─────────────────────────────────────────────╢
+║ autoscaler │ 0.1.0   │ Server, Worker                              ║
+╟────────────┼─────────┼─────────────────────────────────────────────╢
+║ ingress    │ 0.1.0   │ SingletonServer, Server                     ║
+╚════════════╧═════════╧═════════════════════════════════════════════╝
+```
+
+Here is how the previous application configuration looks with an `Ingress` trait attached to the front-end component:
+
+```yaml
+apiVersion: core.oam.dev/v1alpha1
+kind: ApplicationConfiguration
+metadata:
+  name: custom-single-app
+  annotations:
+    version: v1.0.0
+    description: "Customized version of single-app"
+spec:
+  components:
+    - componentName: frontend
+      parameterValues:
+        - name: message
+          value: "Well hello there"
+      traits:
+        - trait:
+            apiVersion: standard.oam.dev/v1alpha1
+            kind: Ingress
+            spec:
+              host: "www.example.com"
+              path: "/"
+```
+
+This example attaches an `ingress` to route traffic from another network (public, perhaps) to the internal service. This configuration shows how the trait is both attached and configured.
+
+An implementation of this would then direct inbound traffic bound for `www.example.com` to the `front-end` component.
+
+### Placing components into application scopes
+
+Up to this point, the frontend component is deployed into the default "root" network and health scopes, as all core workload types are required to be in all core application scope types.
+
+The backend component is an extended workload type and is therefore not automatically deployed into the default "root' application scopes.
+
+In this example, a [network scope](5.application_scopes.md#network-scope) is defined and attached to an SDN. Application Scope instance is created following it's [`ScopeDefinition`](5.application_scopes.md):
+
+```yaml
```

Comment:
oh,  I see. You are creating an instance of scope outside of appConfig. Do we have this concept in v1alpha1?